.github/workflows/test.yml

```
@@ -11,23 +12,23 @@ jobs:
           fetch-depth: 0
-  #     - name: Commit Info
-  #       id: git_info
-  #       run: |
-  #         last_commit_message=$(git log -1 --no-merges --pretty=format:%B)
-  #         echo "last_commit_message=${last_commit_message}"
-  #         echo "last_commit_message<<EOF" >> $GITHUB_OUTPUT
-  #         echo "${last_commit_message}" >> $GITHUB_OUTPUT
-  #         echo "EOF" >> $GITHUB_OUTPUT
+      - name: Commit Info
+        id: git_info
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:3:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)